data.sql

```diff
@@ -16,4 +16,98 @@ VALUES
 INSERT INTO animals
   ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
 VALUES
-  ( 4, 'Devimon', '2017-05-12', 5, 'true', '1' );
\ No newline at end of file
+  ( 4, 'Devimon', '2017-05-12', 5, 'true', '1' );
+
+INSERT INTO animals
+  (ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
+VALUES
+  ( 5, 'Charmander', '2020-02-8', 0, 'false', '-11' );
+
+INSERT INTO animals
+  (ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
+VALUES
+  ( 6, 'Plantmon', '2021-11-15', 2, 'true', '-5.7' );
+
+INSERT INTO animals
+  (ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
+VALUES
+  ( 7, 'Squirtle', '1993-04-2', 3, 'true', '-12.13' );
+
+INSERT INTO animals
+  (ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
+VALUES
```

Comment:
- [ ] Please try to verify your data cause there are some rows that are duplicated so try to remove the duplicate
- [ ] it would be better if you refactor your code into one insert and add all your data like in the example that way your code will be small 
```SQL
/*Insert new records*/
INSERT into animals(name,date_of_birth,escape_attempts,neutered,weight_kg)
VALUES
('Charmander','2020-02-08',0,false,-11),
( 'Plantmon', '2021-11-15', 2, 'true', '-5.7' ),
// other values separate with `,`the last one should followed with `;`
```